core/tests/ts-integration/tests/system.test.ts

```diff
@@ -418,6 +507,51 @@ export interface TransactionData {
     reservedDynamic: ethers.BytesLike;
 }
 
+interface Call {
+    target: string;
+    value: BigNumberish;
+    data: BytesLike;
+}
+
+async function composeL1ToL2Call(
```

Comment:
as discussed please unify this function with the one you copied it from, i.e. import from there etc